src/containers/account/tabs/follows.tsx

```diff
@@ -0,0 +1,71 @@
+import { Pane } from '@cybercongress/gravity';
+import { NoItems } from '../../../components';
+import AvatarIpfs from '../component/avatarIpfs';
+
+type CommunityEntityProps = {
+  items: string[];
+  title: string;
+  noItemsTitle: string;
+};
+
+function CommunityEntity({ items, title, noItemsTitle }: CommunityEntityProps) {
+  return (
+    <Pane marginBottom="20px">
+      <Pane marginBottom="10px" fontSize="20px">
+        {title}
+      </Pane>
+      {items.length > 0 ? (
+        <Pane
+          display="grid"
+          gridTemplateColumns="repeat(auto-fit, minmax(100px, 100px))"
+          width="100%"
+          gridGap="10px"
+        >
+          {items.map((item) => (
+            <AvatarIpfs
+              key={`community_${title}_${item}`}
+              showAddress
+              addressCyber={item}
+            />
+          ))}
+        </Pane>
+      ) : (
+        <NoItems text={noItemsTitle} />
+      )}
+    </Pane>
+  );
+}
+
+type FollowsTabProps = {
```

Comment:
maybe good to split files by components, if we touch them.

Reply:
at next stage